### ISSUE TITLE
Error exit with code 1

### ISSUE THREAD

**Issue by danielkonoplya** [TabularEditor/TabularEditor#1117]
Hello!
I have some problem with tabular editor, i try to deploy model, but i give this error. 
I think the agent doesn't have enough permissions or something else. 
Please tell me what can I do about it. I am using self-hosted agent.
![image](https://github.com/TabularEditor/TabularEditor/assets/31383833/13df66c4-1a54-4d6c-882d-679449ef2df6)

**Comment by otykier** [comment]
Did you specify the full path of the database.json file?

**Comment by danielkonoplya** [comment]
> Did you specify the full path of the database.json file?
 yes, in my working directory

**Comment by otykier** [comment]
Could you share the full TabularEditor.exe command you're using? The error message indicates that Tabular Editor tries to load the file `C:\agent\_work\1\s\database.json`, but no such file exists.

**Comment by danielkonoplya** [comment]
> Could you share the full TabularEditor.exe command you're using? The error message indicates that Tabular Editor tries to load the file `C:\agent\_work\1\s\database.json`, but no such file exists.

I ran it locally and it works, but it doesn't work in the pipeline
![image](https://github.com/TabularEditor/TabularEditor/assets/31383833/e9c51c4c-fe04-4426-92bc-183af783f1a0)

**Comment by danielkonoplya** [comment]
> Could you share the full TabularEditor.exe command you're using? The error message indicates that Tabular Editor tries to load the file `C:\agent\_work\1\s\database.json`, but no such file exists.

I ran it locally and it works, but it doesn't work in the pipeline
![image](https://github.com/TabularEditor/TabularEditor/assets/31383833/7dc6344f-9bdf-4aaa-b72a-e8d2486b236e)

**Comment by otykier** [comment]
Indeed, it must be some permission setting of the service account used to run the agent. This is unfortunately outside of my area of expertise. You may have more luck asking Microsoft support.

**Comment by danielkonoplya** [comment]
> Indeed, it must be some permission setting of the service account used to run the agent. This is unfortunately outside of my area of expertise. You may have more luck asking Microsoft support.

Ok thanks
